src/main/java/edu/unc/lib/boxc/migration/cdm/InitializeProjectCommand.java

```diff
@@ -87,48 +90,60 @@ public Integer call() throws Exception {
             log.error("Unable to read application configuration", e);
             return 1;
         }
-        var cdmEnvConfig = config.getCdmEnvironments().get(cdmEnvId);
 
         Path currentPath = parentCommand.getWorkingDirectory();
         String projDisplayName = projectName == null ? currentPath.getFileName().toString() : projectName;
         String collId = cdmCollectionId == null ? projDisplayName : cdmCollectionId;
+        String username = System.getProperty("user.name");
 
-        // Retrieve field information from CDM
-        CdmFieldInfo fieldInfo;
-        try {
-            fieldService.setCdmBaseUri(cdmEnvConfig.getHttpBaseUrl());
-            fieldInfo = fieldService.retrieveFieldsForCollection(collId);
-        } catch (IOException | MigrationException e) {
-            log.error("Failed to retrieve field information for collection in project", e);
-            outputLogger.info("Failed to retrieve field information for collection {} in project {}:\n{}",
-                    collId, projDisplayName, e.getMessage());
-            return 1;
-        }
+        if (newProject != null && !newProject.isBlank()) {
+            // Instantiate the project
+            try {
```

Comment:
Might help with readability to have separate `initCdmProject` and `initFilesProject` private methods (which returns an Integer)